docs/DeveloperGuide.md

```diff
@@ -167,53 +168,17 @@ Classes used by multiple components are in the `seedu.taassist.commons` package.
 
 ## **Implementation**
 
-### Creating and deleting module classes
-
-Module class information is stored as `ModuleClass` objects, which captures the name of the module class as well as the sessions created for the module class. When the user creates a module class, the program creates a new `ModuleClass` object with the module class name and adds it to the collection of `ModuleClass` objects previously created, which is managed by the `TaAssist` class.
-
-When deleting module classes, all students previously assigned to those module classes will have the module classes unassigned from them, before the `ModuleClass` objects corresponding to those module classes are removed from the collection captured by `TaAssist`.
-
-The following methods in `TaAssist` manages the adding and deleting of module classes from the collection:
+This section describes some noteworthy details on how certain features are implemented along with explanations
+for why certain functions are implemented in such a manner.
 
-- `TaAssist#addModuleClass(ModuleClass moduleClass)` - Adds the provided module class to the list of module classes created.
-- `TaAssist#removeModuleClass(ModuleClass moduleClass)` - Removes the provided module class from the list of module classes created.
-
-### Assigning students to module classes
+### Assigning students to a class
 <img src="images/AssignCommandSequenceDiagram.png" width="700" />
 
-Each student object contains a collection of `StudentModuleData` where module classes and the grades the student obtained for the sessions of the module classes are stored. When the user assigns students to a module class, a new `StudentModuleData` object is created and added to the collection for each student.
```

Comment:
I think this was added by @Bubbl3T, and we probably need to keep these parts (i.e. Creating and deleting module classes and Assigning students to module classes so on and so for)

Reply:
This issue has been resolved. Please do check again :)